src/index.js

```diff
@@ -1,23 +1,17 @@
 import propTypes from 'prop-types'
-
 import { arrayWithLength } from './propTypes/arrayWithLength.js'
 import { conditional } from './propTypes/conditional'
 import { instanceOfComponent } from './propTypes/instanceOfComponent.js'
 import { mutuallyExclusive } from './propTypes/mutuallyExclusive.js'
 import { requiredIf } from './propTypes/requiredIf.js'
 
-propTypes.arrayWithLength = arrayWithLength
-propTypes.conditional = conditional
-propTypes.instanceOfComponent = instanceOfComponent
-propTypes.mutuallyExclusive = mutuallyExclusive
-propTypes.requiredIf = requiredIf
-
-export {
+const dhis2PropTypes = {
+    ...propTypes,
     arrayWithLength,
     conditional,
     instanceOfComponent,
     mutuallyExclusive,
     requiredIf,
 }
 
-export default propTypes
+export default dhis2PropTypes
```

Comment:
LGTM, one minor thing, don't care too much, so I approve

I'd like to see this:

```js
propTypes.arrayWithLength = arrayWithLength
propTypes.conditional = conditional
propTypes.instanceOfComponent = instanceOfComponent
propTypes.mutuallyExclusive = mutuallyExclusive
propTypes.requiredIf = requiredIf

export * from 'prop-types'
```

This way we'd always expose whatever `'prop-types'` exposes (let's say they have different default and named exports). It's very unlikely but possible.

Reply:
I disagree, I prefer having control of the default export.

Reply:
We've talked about it here: https://dhis2.slack.com/archives/CPGUFVC56/p1585823170090400, and decided to leave it for now.